addons/mail/views/assets.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <data>
-        <template id="_assets_primary_variables" inherit_id="web._assets_primary_variables">
+        <template id="_assets_helpers" inherit_id="web._assets_helpers">
```

Comment:
So basically this PR should be 3 lines: this one here (new) + corresponding closing tag and new line after.

With commit explaining that it is necessary to merge introducing new assets in 13.4.